pkg/epp/scheduling/framework/plugins/multi/prefix/plugin.go

```diff
@@ -239,6 +248,45 @@ func (m *Plugin) matchLongestPrefix(ctx context.Context, hashes []BlockHash) map
 	return res
 }
 
+// StartPodActiveWatcher starts a goroutine that watches for active pods.
+func (m *Plugin) StartPodActiveWatcher(ctx context.Context, handle plugins.Handle) {
+	logger := log.FromContext(ctx).V(logutil.VERBOSE)
+
+	ticker := time.NewTicker(PodActiveCheckInterval)
+	defer ticker.Stop()
+
+	podLastSeen := make(map[ServerID]time.Time)
```

Comment:
I am thinking of a much simpler implementation: iterate over current pods in the `podToLRU` in the indexer, and if that pod doesn't exist in `ActivePods`, remove. And this approach does not need a `PodInactivityTimeout`.